index.js

```diff
@@ -151,6 +151,10 @@ module.exports = function (newrelic, opts) {
 	}
 
 	return async function koaNewrelic(ctx, next) {
+		if (opts.preSetTransaction) {
+			setTransactionName(ctx.method, ctx.path);
```

Comment:
this will explode ur newrelic log, since it will treat each path separately, such as
```
GET /tracking/1
GET /tracking/2
GET /tracking/3
...
```
instead of 
```
GET /tracking/:id
```